clients/CMakeLists.txt

```diff
@@ -134,9 +136,11 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
   find_package(Threads REQUIRED)
 
   # Look for openmp config in ROCm install to populate openmp_LIB_DIR and openmp_LIB_INSTALL_DIR
-  find_package(OpenMP CONFIG PATHS "${HIP_CLANG_ROOT}/lib/cmake")
+  if( CMAKE_CXX_COMPILER_ID MATCHES "Clang"  )
+    find_package(OpenMP CONFIG PATHS "${HIP_CLANG_ROOT}/lib/cmake")
+  endif()
 
-  if (TARGET OpenMP::omp)
+  if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND TARGET OpenMP::omp)
```

Comment:
This "clang check" here may be a bit repetitive as the only way OpenMP::omp is set if line 140 is executed.

Reply:
> This "clang check" here may be a bit repetitive as the only way OpenMP::omp is set if line 140 is executed.

Sure but like to be clear when I expect to going into the if